CONTRIBUTING.md

```diff
@@ -1 +1,172 @@
-This is a place holder
+# Contributing to Quail
+
+Thanks for your interest in contributing! We are excited to have you join our team!
+
+The following is a set of guidelines for contributing to Quail [Ihme Group Organization](https://github.com/IhmeGroup/quail) on GitHub. Feel free to propose modifications to this document via a pull request as we value others contributions. 
+
+#### Table Of Contents
+
+[Getting Started](#getting-started)
+  * [Video Tutorials](#some-video-tutorials)
+  * [Interfaces](#neat-but-how-do-i-interface-with-quails-packages)
+
+[Style Guide](#style-guide)
+
+[Branch naming conventions](#branch-naming-conventions)
+## Getting Started
+
+Quail is a lightweight discontinuous Galerkin code written with the intention of being useful for teaching and prototyping. Quail is made up of a series of Python packages. These are all located in the `src` directory. They include the following:
+
+* [solver](https://github.com/IhmeGroup/quail/tree/main/src/solver) - which contains modules for various solvers (DG and ADER-DG). This is where the primary `solve` function is located which governs the loop around which iterations occur.
```

Comment:
"governs the loop around which iterations occur" sounds a little awkward

maybe something simpler like "advances the solution in time"